docs/source/user/threading-layer.rst

```diff
@@ -26,9 +26,10 @@ There are three threading layers available and they are named as follows:
 * ``workqueue`` -A simple built-in work-sharing task scheduler.
 
 In practice, the only threading layer guaranteed to be present is ``workqueue``.
-The ``omp`` layer requires the presence of a suitable OpenMP runtime library.
-The ``tbb`` layer requires the presence of Intel's TBB libraries, these can be
-obtained via the conda command::
+The ``omp`` layer requires the presence of a suitable (2.0 or greater on
+Windows, 3.0 or greater otherwise) OpenMP runtime library.  The ``tbb`` layer
```

Comment:
Should probably mention the constraint in the installation/dependency list too? https://numba.readthedocs.io/en/stable/user/installing.html#dependency-list

Reply:
I've added them there - the descriptions there were pretty vague already, but I believe I've improved it a little with some more specific info.